### DISCUSSION TITLE
Button to go -85 seconds backwards

### DISCUSSION THREAD

**Discussion by pavlidisnick** [aniyomiorg/aniyomi#551] [Ideas]
Honestly its the only thing that is missing. For me the app is a solid 9.5/10

**Comment by Pervases** [comment]
What do you need it for anyway?

**Reply by pavlidisnick** [reply]
for the same reason you need the forward button

**Reply by Pervases** [reply]
The reason for **+85** is because that's the usual length of an anime's OP. You use it to skip the OP.

I haven't thought about it but I'm curious now. Is there any other use for it?

**Reply by pavlidisnick** [reply]
sure but its good to have controls on a media player anyways, if something happened like a telephone etc and you pause it too late

**Reply by Pervases** [reply]
I might sound like I'm arguing with you but I'm not. If anything, I am just trying to help.

-85 seconds is a long rewind, 1 minute and 25 seconds. If you just need to rewind you can just use **Double tap to skip length**. You can also adjust the length to your liking, 5, 10, 20, 30 seconds.

At first, my OCD was firing due to the lack of symmetry on +85 controls, I got over it tho cause I realized it has no practical use.

**Reply by pavlidisnick** [reply]
oh man I thought I tried it before to double tap and it didn't rewind but I just tried it and it works...thanks! so ye now I see no use
